IntersonArrayDeviceRF.hxx

```diff
@@ -118,14 +121,14 @@ public:
       {
       Stop();
       hwControls.GetFrequency( frequencies );
-      success = hwControls.SetFrequencyAndFocus( frequencyIndex, focusIndex,
-        steering );
+      if( hwControls.SetFrequencyAndFocus( frequencyIndex, focusIndex,
```

Comment:
Is this the preferred way instead of setting the boolean success directly? Kitware style?

Reply:
Nah - just a personal style.   I got carried away.  I can revert this.